diff --git a/webots_ros2_core/webots_ros2_core/webots_node.py b/webots_ros2_core/webots_ros2_core/webots_node.py
index 42758a7..977f2fc 100644
--- a/webots_ros2_core/webots_ros2_core/webots_node.py
+++ b/webots_ros2_core/webots_ros2_core/webots_node.py
@@ -41,8 +41,8 @@ class WebotsNode(Node):
 
     def __init__(self, name, args=None):
         super().__init__(name)
-        self.declare_parameter('synchronization',
-                               Parameter('synchronization', Parameter.Type.BOOL, False))
+        self.declare_parameter('synchronization', False)
+
         parser = argparse.ArgumentParser()
         parser.add_argument('--webots-robot-name', dest='webotsRobotName', default='',
                             help='Specifies the "name" field of the robot in Webots.')
